neo/Network/RPC/RpcServer.cs

```diff
@@ -236,6 +236,20 @@ private JObject Process(string method, JArray _params)
                         string address = _params[0].AsString();
                         return ValidateAddress(address);
                     }
+                case "plugins":
+                    {
+                        return new JArray(Plugin.Plugins
+                            .OrderBy(u => u.Name)
+                            .Select(u =>
+                            {
+                                var obj = new JObject();
+                                obj["name"] = u.Name;
+                                obj["version"] = u.Version.ToString();
+
```

Comment:
```
obj["interfaces"] = new JArray(u.GetType().GetInterfaces().Select(p => p.Name).Where(p => p.EndsWith("Plugin")).Select(p => (JObject)p));
```

Reply:
It needs to show the plugin name from the plugin object itself I think, because someone can have a project that includes Neo.dll that has lots of plugins defined in it and not loaded as individual dlls.